tez-ui/pom.xml

```diff
@@ -374,7 +374,7 @@
             </goals>
             <configuration>
               <nodeVersion>${nodeVersion}</nodeVersion>
-              <yarnVersion>v0.21.3</yarnVersion>
+              <yarnVersion>v1.6.0</yarnVersion>
```

Comment:
is this related to async upgrade?

Reply:
please confirm this one

Reply:
@abstractdog I confirm this change

Reply:
I mean, this looks like a major version change of yarn, can you please let me know how is this related to the CVE patch (or is it necessary)?
I prefer doing version changes in separate patches if they're not closely related

Reply:
@abstractdog The yarnVersion was changed since the selective dependency feature is only supported in yarn 1.0 and above. And yarnVersion of 1.6.0 is compatible with frontend maven plugin 1.8.0. That's why I did this change.

Reply:
cool, thanks for clarifying @amanraj2520 !